src/backend/executor/nodeSeqscan.c

```diff
@@ -87,8 +101,17 @@ SeqNext(SeqScanState *node)
 	/*
 	 * get the next tuple from the table
 	 */
-	if (table_scan_getnextslot(scandesc, direction, slot))
+	while (table_scan_getnextslot(scandesc, direction, slot))
+	{
+		if (TupIsNull(slot))
+			return slot;
```

Comment:
Will it be true?

Reply:
It seems that slot is never NULL here, so `Assert(!TupIsNull(slot));` is better or remove them.

Reply:
fix with https://github.com/apache/cloudberry/pull/724/commits/99eabb224f89dcdfd7117ef37a5986ab8001abdf